zone/client.cpp

```diff
@@ -6541,7 +6540,7 @@ void Client::Doppelganger(uint16 spell_id, Mob *target, const char *name_overrid
 		NPC* swarm_pet_npc = new NPC(
 				(npc_dup!=nullptr)?npc_dup:npc_type,	//make sure we give the NPC the correct data pointer
 				0,
-				GetPosition() + glm::vec4(swarmPetLocations[summon_count], 0.0f, 0.0f),
+				GetPosition() + glm::vec4(swarmPetLocations[summon_count - 1], 0.0f, 0.0f),
```

Comment:
So if we're doing `>= 0` for the loop here, when we get to `0` we'd be accessing index `-1` here, right? Or am I reading this incorrectly.

Reply:
Correct, I meant to only do one of those two changes (I was playing with doing >= or -1, it shouldn't have been both)

Reply:
👍